x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_card/product_card.tsx

```diff
@@ -30,6 +32,27 @@ interface IProductCard {
 
 export const ProductCard: React.FC<IProductCard> = ({ product, image }) => {
   const { http } = useValues(HttpLogic);
+  const {
+    config: { host },
+  } = useContext(KibanaContext) as IKibanaContext;
+
+  const LAUNCH_BUTTON_TEXT = i18n.translate(
+    'xpack.enterpriseSearch.overview.productCard.launchButton',
+    {
+      defaultMessage: `Launch {productName}`,
```

Comment:
Super quick: we don't need template literals here
```suggestion
      defaultMessage: 'Launch {productName}',
```